locale/en/download/releases.md

```diff
@@ -1,7 +1,20 @@
 ---
 layout: download-releases.hbs
 title: Previous Releases
-iojs:
-  intro: "Releases 1.x through 3.x were called \"io.js\" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases."
 modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. It used to be stored as hex value in earlier versions, but is now represented as an integer."
 ---
+
+### io.js &amp; Node.js
+Releases 1.x through 3.x were called "io.js" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases.
+
+<div class="highlight-box">
+    Looking for latest release of a version branch?
+
+    <ul class="list-divider-pipe">
+        <li><a href="https://nodejs.org/dist/latest-v6.x/">Node.js 6.x</a></li><li><a href="https://nodejs.org/dist/latest-v5.x/">Node.js 5.x</a></li>
```

Comment:
If you add a new line here, after the first `<li>`, we can probably avoid to add the padding in `_lists.styl`.